Dockerfile

```diff
@@ -1,28 +1,18 @@
-FROM keydonix/parity-instantseal
+FROM keydonix/parity-instantseal-node8
 # TODO: use digest
 
-WORKDIR /build
-
 # TODO: vendor
 RUN apt-get update && apt-get -y install software-properties-common git make && \
 	add-apt-repository ppa:ethereum/ethereum && \
 	apt-get update && \
 	apt-get install -y solc
 
-RUN git clone https://github.com/makerdao/sai.git && \
-	( cd sai && \
-	git checkout 56eed66bb42a485ff0819f4ca227f615b1eb5320 && \
-	git submodule update --init --recursive )
-
-RUN mkdir artifacts && mkdir tools && cd tools && \
-	git clone https://github.com/dapphub/dapp && \
-	(cd dapp && git checkout a426596705be4dfcdd60e7965163453574459dcf) && \
-	make install -C dapp prefix=/build/artifacts
-
-ENV PATH="/build/artifacts/bin:${PATH}"
-
-WORKDIR /build/sai
+COPY . /build/maker-docker-poa
 
-RUN dapp build
+WORKDIR /build/maker-docker-poa
 
-WORKDIR /
+RUN npm install
+RUN npx tsc && \
+        ./fetch-sai-contracts.sh \
```

Comment:
Missing an `&&` here?

Reply:
You're lookin' at the old PR's, i'm closing these and it is fixed in the new